travis_script.sh

```diff
@@ -57,6 +57,21 @@ do
     popd
 done
 
+declare -a ANALYZE_PROJECT_NAMES=(
```

Comment:
Are there likely to be other apps that require this type of check? If not, don't be afraid to just do it directly rather than use a loop.

Reply:
True, most likely not. I'll move it out to just do it directly.